internal/captain/command.go

```diff
@@ -655,8 +654,7 @@ func (c *Command) cobraExecHandler(cobraCmd *cobra.Command, args []string) (rerr
 		c.out.Notice(locale.Tr("unstable_command_warning"))
 		return nil
 	} else if c.out.Type().IsStructured() && !c.structuredOutput {
-		c.out.Error(locale.NewInputError("err_no_structured_output", "", string(c.out.Type())))
-		return nil
+		return locale.NewInputError("err_no_structured_output", "", string(c.out.Type()))
```

Comment:
I think this was a bad copy paste. Before this we were catching the error coming from the output mediator, and not from the captain runner, which was the intended mechanic.